src/app/profile/update/update.component.html

```diff
@@ -1,170 +1,192 @@
+        <a href="javascript:void(0)" (click)="sendEmailVerificationLink()" class="alert-link pull-right">Resend verification link</a>
+      </div>
+    </div>
+  </div>
   <div class="row profile-edit">
-    <!-- left column -->
-    <div class="col-xs-12 col-sm-5 col-sm-offset-1">
-      <h1>Edit Profile</h1>
-      <form role="form" #profileForm="ngForm">
-        <div class="form-group">
-          <p class="help-block">The fields marked with <span class="required-pf">*</span> are required.</p>
-        </div>
-        <div class="form-group">
-          <label for="fullName" class="control-label">Name</label>
-          <div [ngClass]="{'has-error': fullNameInvalid}">
-            <input type="text" class="form-control" id="fullName" name="fullName"
-                    placeholder="Enter your full name"
-                    [maxLength]="80"
-                    [(ngModel)]="fullName">
-          </div>
-        </div>
-        <div class="form-group">
-          <label for="email" class="control-label required-pf email-alignment-label">Email</label>
-          <div [ngClass]="{'has-error': emailInvalid}" class="email-alignment">
-            <input type="email" class="form-control" id="email" name="email" #_email
-                    placeholder="name@gmail.com"
-                    [maxLength]="255"
-                    [(ngModel)]="email"
-                    (change)="validateEmail()">
-            <div class="radio email-alignment_settings">
-              <label>
-                <input type="radio" name="emailVisibility" id="emailPrivate"
-                        [(ngModel)]="emailPrivate"
-                        [value]="true"
-                        [checked]="emailPrivate">
-                Private
-              </label>
-              <label class="margin-left-30">
-                <input type="radio" name="emailVisibility" id="emailPublic"
-                        [(ngModel)]="emailPrivate"
-                        [value]="false"
-                        [checked]="!emailPrivate">
-                Public
-              </label>
+    <div class="col-xs-4 col-md-1">
+        <div *ngIf="(imageUrl !== undefined && imageUrl.length !== 0); then showImageUrl else addImageUrl"></div>
+      <ng-template #showImageUrl>
+        <img src="{{imageUrl}}" class="img-responsive profile-img icon-user">
+      </ng-template>
+      <ng-template #addImageUrl>
+        <img src="../../../assets/images/profile-user.png" class="img-responsive profile-img-none icon-user">
+      </ng-template>
+    </div>
 
+    <div class="col-xs-12 col-md-11">
+      <div class="row padding-bottom-15">
+        <p class="col-xs-12"><label for="previewImg" class="control-label padding-left-none">Profile photo</label></p>
+        <p class="col-md-2 link-profile-photo">
+          <a href="javascript:void(0)" (click)="linkGravatarImageUrl()">
+            <span class="fa fa-gravatar"></span> Import from Gravatar
```

Comment:
This is wrapping, when viewed through the badger link.

![screen shot 2018-04-26 at 1 34 48 pm](https://user-images.githubusercontent.com/4032718/39322064-aa0c6ad2-4956-11e8-8977-305b235f0337.png)